### ISSUE TITLE
Feature Request: Latent Classes

### ISSUE THREAD

**Issue by yangyang2000** [timothyb0912/pylogit#11]
Any chance of getting latent class models added to this package? Something similar to the GMNL package in R.

**Comment by timothyb0912** [comment]
Yeah I'll add latent classes at some point. Not necessarily soon though because I'm working on my dissertation.

Feel free to open a pull request where you take a stab at it though!
